src/harness/fourslash.ts

```diff
@@ -1740,31 +1722,25 @@ namespace FourSlash {
                 }

```

Comment:
Do you know why above in `if (tyhis.enableFormatting)` we update `offset` but not `currentCaretPosition`?
Also, is `fixCaretPosition` still necessary if we update it correctly?